doc/release-notes.md

```diff
@@ -12,3 +12,24 @@ presence in blocks. This wasn't being used for more than an expensive
 sanity check. Since 5.1.0, these are no longer stored. When loading a
 5.1.0 wallet into an older version, it will automatically rescan to avoid
 failed checks.
+
+Account recovery from emergency recovery phrase
+-----------------------------------------------
+Accounts can now be recovered using the emergency recovery phrase. This is done
+by starting zcashd with the "-recoverwallet" flag, and providing a path to the
+wallet recovery file. Below is an example recovery file to illustrate the
+expected format:
+
+recovery_phrase="abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon abandon art"
+language="English"
+fingerprint="examplefingerprint"
+num_accounts=3
+
+Only the `recovery_phrase=` line is mandatory. If there is no provided language,
+zcashd will default to English. If there is no provided num_accounts, zcashd
+will default to `num_accounts=1` (i.e. it will recover only account 0).
+In particular, if we are recovering from a wallet dump taken before v5.3.0,
+only account 0 will be recovered.
```

Comment:
```suggestion
Only the `recovery_phrase=` line is mandatory. If there is no provided language,
zcashd will default to English. Keys will be recovered for each account identifier
less than `num_accounts`. If there is no value provided for `num_accounts`, 
zcashd will default to `num_accounts=1` (i.e. it will recover only account 0).
In particular, if we are recovering from a wallet dump taken before v5.3.0,
only account 0 will be recovered.
```